### ISSUE TITLE
Cant interpolate aria-label 

### ISSUE THREAD

**Issue by neofuture** [angular/angular#33913]
# 🐞 bug report

### Description

Unable to use aria-label="{{ var }}"

## 🔬 Minimal Reproduction

This works

<code><pre>
<input id="searchBox" type="text" placeholder="{{ search }}" [(ngModel)]="searchTerm"
         (keyup)="checkKeys($event)"
         (keydown)="preventCursor($event)"
         (blur)="closeSearchDebounce()"
         (focus)="checkKeys($event)"
         (click)="consolelog($event)"
         [ngClass]="objectKeys(searchHits).length > 0 ? 'results' : ''"
         [ngStyle]="{'padding-left': searchTerm.length > 0 ? '4px' : '24px'}"
         autocapitalize="off" autocomplete="off" autocorrect="off" spellcheck="false"
         aria-label="S">
</pre></code>

this does not

<code><pre>
<input id="searchBox" type="text" placeholder="{{ search }}" [(ngModel)]="searchTerm"
         (keyup)="checkKeys($event)"
         (keydown)="preventCursor($event)"
         (blur)="closeSearchDebounce()"
         (focus)="checkKeys($event)"
         (click)="consolelog($event)"
         [ngClass]="objectKeys(searchHits).length > 0 ? 'results' : ''"
         [ngStyle]="{'padding-left': searchTerm.length > 0 ? '4px' : '24px'}"
         autocapitalize="off" autocomplete="off" autocorrect="off" spellcheck="false"
         aria-label="{{ search }}">
</pre></code>

difference Is 

<code>aria-label="{{ search }}"</code>
## 🔥 Exception or Error
<pre><code>
ERROR in src/app/components/search-bar/search-bar.component.html:11:10 - error TS8002: Can't bind to 'aria-label' since it isn't a known property of 'input'.

11          aria-label="{{ search }}">
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  src/app/components/search-bar/search-bar.component.ts:9:16
    9   templateUrl: './search-bar.component.html',
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component SearchBarComponent.
</code></pre>


## 🌍  Your Environment

**Angular Version:**
<pre><code>
Angular CLI: 9.0.0-rc.2
Node: 12.13.0
OS: darwin x64
Angular: 9.0.0-rc.2
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router, service-worker

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.900.0-rc.2
@angular-devkit/build-angular     0.900.0-rc.2
@angular-devkit/build-optimizer   0.900.0-rc.2
@angular-devkit/build-webpack     0.900.0-rc.2
@angular-devkit/core              9.0.0-rc.2
@angular-devkit/schematics        9.0.0-rc.2
@angular/cdk                      8.2.3
@angular/pwa                      0.803.19
@ngtools/webpack                  9.0.0-rc.2
@schematics/angular               8.3.19
@schematics/update                0.900.0-rc.2
rxjs                              6.5.3
typescript                        3.6.4
webpack                           4.41.2
</code></pre>

**Comment by neofuture** [comment]
[attr.aria-label] is the only answer ?

**Comment by neofuture** [comment]
My Bad should have read the docs
